packages/app/src/messages/components/delete-message-modals/delete-message-modals.component.tsx

```diff
@@ -3,88 +3,77 @@
-}
 
 const messages = defineMessages({
-  deleteMessageModalTitle: {
-    id: "module.messages.deleteMessageModalTitle",
-  },
-  deleteMessageConfirmationModalBody: {
-    id: "module.messages.deleteMessageConfirmationModalBody",
-  },
-  deleteMessageConfirmationModalCancel: {
-    id: "module.messages.deleteMessageConfirmationModalCancel",
-  },
-  deleteMessageConfirmationModalDelete: {
-    id: "module.messages.deleteMessageConfirmationModalDelete",
-  },
-
-  deleteMessageSuccessPopupInfo: {
-    id: "module.messages.deleteMessageSuccessPopupInfo",
-  },
-
-  deleteMessageFailureModalSubtitle: {
-    id: "module.messages.deleteMessageFailureModalSubtitle",
+  deletedMessageInfo: { id: "module.messages.deletedMessageInfo" },
+  deleteModalTitle: { id: "module.messages.deleteMessageModalTitle" },
+  deleteModalBody: { id: "module.messages.deleteMessageModalBody" },
+  deleteModalCancel: { id: "module.messages.deleteMessageModalCancel" },
+  deleteModalAction: { id: "module.messages.deleteMessageModalAction" },
+  deletingModalTitle: { id: "module.messages.deletingMessageModalTitle" },
+  deletingModalSubtitle: { id: "module.messages.deletingMessageModalSubtitle" },
+  deleteModalErrorTitle: { id: "module.messages.deleteMessageModalErrorTitle" },
+  deleteModalErrorSubtitle: {
+    id: "module.messages.deleteMessageModalErrorSubtitle",
   },
 })
 
-const DeleteMessageModals: FunctionComponent<Props> = ({
-  messageDeletingState,
-  openDeleteMessageConfirmation,
-  onMessageRemove,
-  hideDeleteErrorModal,
-  hideConfirmationModal,
+const DeleteMessageModals: FunctionComponent<DeleteMessageModalProps> = ({
+  error,
+  deleting,
+  deletingInfo,
+  deletingConfirmation,
+  onDelete,
+  onCloseDeletingModal,
+  onCloseDeletingErrorModal,
 }) => {
   return (
     <>
-      {openDeleteMessageConfirmation && (
+      {deletingConfirmation && (
         <DeleteConfirmationModal
           testId={DeleteMessageModalsTestIds.DeleteMessageConfirmation}
-          open={openDeleteMessageConfirmation}
-          info={messages.deleteMessageConfirmationModalBody}
-          onActionButtonClick={onMessageRemove}
-          onCloseButton={hideConfirmationModal}
-          cancelButtonLabel={intl.formatMessage(
-            messages.deleteMessageConfirmationModalCancel
-          )}
-          actionButtonLabel={intl.formatMessage(
-            messages.deleteMessageConfirmationModalDelete
-          )}
-          titleLabel={intl.formatMessage(messages.deleteMessageModalTitle)}
+          open={deletingConfirmation}
+          titleLabel={intl.formatMessage(messages.deleteModalTitle)}
+          info={messages.deleteModalBody}
+          onActionButtonClick={onDelete}
+          onCloseButton={onCloseDeletingModal}
+          cancelButtonLabel={intl.formatMessage(messages.deleteModalCancel)}
+          actionButtonLabel={intl.formatMessage(messages.deleteModalAction)}
+        />
+      )}
+
+      {deleting && !error && (
```

Comment:
According to ACC, Loader Modal should not be applied for deleting message (the loading state for deleting message is indicated by adding opacity to an affected message)

Reply:
https://github.com/mudita/mudita-center/pull/827/commits/65fc2a6f831ca12bbabffd8757fe96501fea9427